index.ts

```diff
@@ -120,7 +128,7 @@ async function init() {
 		},
 	});
 
-	if ('downloadUrl' in parsedPath) {
+	if ('downloadUrl' in parsedPath && !isPrivate) {
 		updateStatus('Downloading the entire repository directly from GitHub');
```

Comment:
See below. Why does this not work on private repos? If `parsedPath.downloadUrl` exists, it should work

Reply:
That issue is about the token, not download URLs